src/SortableList.js

```diff
@@ -222,10 +229,10 @@ export default class SortableList extends Component {
       if (rowsLayouts) {
         if (horizontal) {
           location.x = nextX;
-          nextX += rowsLayouts[key].width;
+          nextX += rowsLayouts[key] ? rowsLayouts[key].width : 0;
```

Comment:
do we really need this check?

Reply:
Yes - I ran into crashes using react-navigation when this component was not currently on the screen but being updated. I probably could have made a separate PR for that. Let me know if you'd prefer that.

Reply:
yea, separate pr will be better

Reply:
Coming right up.

Reply:
Moved fix over to #93